app/views/search/index.html.erb

```diff
@@ -1,11 +1,28 @@
 
 <div class="row row-content">
   <section>
-    <div id="collection_search_criteria">
-      <%= render partial: 'collection_search_criteria' %>
-    </div>
-    <div id="collection_search_results">
-      <%= render partial: 'collection_search_results' %>
+    <div id="search-criteria">
+      <%= render partial: 'search_criteria' %>
```

Comment:
When using partials we should pass variables in using `locals` rather than expecting instance variables to exist. (I know this is how collections does it so 1. Update both if its not a massive piece of work **or** 2. improve this and we'll continue to work this type of thing (collections) in the future as long as it pertains to a ticket)